static/locales/en-US.yaml

```
@@ -664,6 +664,7 @@ Settings:
   Parental Control Settings:
     Parental Control Settings: Parental Control
     Hide Unsubscribe Button: Hide Unsubscribe Button
+    Hide Uploader: Hide Uploader
```

```suggestion
    Hide Uploader: Hide Uploader on watch page
```
This part is required otherwise it isnt clear where this is applied

---

`Hide Uploader on Watch Page`
otherwise it looks strange :)